lib/ecto/migration.ex

```diff
@@ -647,7 +647,10 @@ defmodule Ecto.Migration do
   ## Options
 
     * `:primary_key` - when `false`, a primary key field is not generated on table
-      creation.
+      creation. Alternatively, a keyword list in the same style of the
```

Comment:
Do we really need this? Why can't one set `primary_key: false` + manually setting the primary key column?

Reply:
See the discussion in the mailing list. If you want to change the defaults dynamically, you would have to reimeplement the whole `create` **macro**.